src/main/kotlin/com/example/routes/tasks/taskRouter.kt

```diff
@@ -137,7 +137,8 @@ fun Route.tasks(context: DSLContext) {
                         var count = 0
```

Comment:
```suggestion
                        var totalPoints = 0
```

Reply:
Fixed with [c2e3db6](https://github.com/we-huddle/server/pull/43/commits/c2e3db6ee51cca4058328af208f9ed8b5a45a00e) 
cc: @Gravewalker666